src/Provider/ShopifyStore.php

```diff
@@ -3,9 +3,12 @@
 namespace Multidimensional\OAuth2\Client\Provider;
 
 use League\OAuth2\Client\Provider\ResourceOwnerInterface;
+use League\OAuth2\Client\Tool\ArrayAccessorTrait;
 
 class ShopifyStore implements ResourceOwnerInterface
 {
+    use ArrayAccessorTrait;
```

Comment:
Simply adding a `use` statement is incomplete. You'd want to change each of the accesses for `$this->response`:

```php
return $this->getValueByKey($this->response, 'shop.id');
```